src/test/java/com/connexta/store/CustomTestRestTemplate.java

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
Created class where new functionality can be added TestRestTemplate to reduce boiler plate code.
It's use in this PR is to automatically add the "Accept-Version" header to requests.